705.py

```diff
@@ -0,0 +1,50 @@
+# N.B.
+# We should go back visit linkedlist and BST solutions.
```

Comment:
Great! Looks like you were already aware of what we need to do to fully digest this design hash set problem.